airbyte-integrations/connectors/source-mailjet-mail/source_mailjet_mail/manifest.yaml

```diff
@@ -83,7 +83,7 @@ definitions:
     $parameters:
       name: "message"
       primary_key: "ID"
-      path: "/message"
+      path: "/message?ShowSubject=true"
```

Comment:
This doesn't impact in the schema?

Reply:
I don't think so, because ``subject`` already exists in the schema but it always null